synth.py

```diff
@@ -21,21 +21,27 @@
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICGenerator()
+gapic = gcp.GAPICMicrogenerator()
 common_templates = gcp.CommonTemplates()
 
 versions = ['v1', 'v1beta1']
 
 for version in versions:
-    library = gapic.node_library('texttospeech', version)
+    library = gapic.typescript_library(
+        'texttospeech', 
+        generator_args={
+            "grpc-service-config": f"google/cloud/texttospeech/{version}/texttospeech_grpc_service_config.json"
+            },
+        proto_path=f'/google/cloud/texttospeech/{version}',
+        version=version)
 
     # skip index, protos, package.json, and README.md
     s.copy(
         library,
-        excludes=['package.json', 'README.md', 'src/index.js'],
+        excludes=['package.json', 'README.md', 'src/index.ts'],
```

Comment:
Curiosity, why are we ignoring README here?

Reply:
I'm not very familiar with synthtool, but I steal this from the `synth.py` in `nodejs-text-to-speech` :) not sure if there is a special reason to skip these files. Thanks

Reply:
You know what, I think this is on purpose. Ignore me :)